src/components/Calculator.js

```diff
@@ -0,0 +1,185 @@
+import PropTypes from 'prop-types';
+
+function Calculator() {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        width: 400,
+        height: 'auto',
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          height: 50,
+          flex: 1,
+        }}
+      >
+        <Display numberParent={0} />
```

Comment:
- [ ] kindly consider sticking to the given design by placing this `numberParent` to the right of the grey container to make sure that your design implements all the design guideline
- your design 👇 
<img width="397" alt="Screen Shot 2023-07-04 at 22 41 45" src="https://github.com/Kidd254/Math-Magicians-React-App/assets/60146030/e490bea2-2e47-4b34-bb63-4e2da8fe28da">
- expected design 👇 
<img width="302" alt="Screen Shot 2023-07-04 at 22 42 45" src="https://github.com/Kidd254/Math-Magicians-React-App/assets/60146030/b8a99ffe-85e4-4f88-a3a4-fffbcdf827b5">